datasync/chaindatafetcher/kafka/kafka_test.go

```diff
@@ -165,10 +165,11 @@ func (s *KafkaSuite) subscribeData(topic, groupId string, numTests int, handler
 	}()
 
 	// wait for all data to be consumed
-	timeout := time.NewTimer(3 * time.Second)
+	timeout := time.NewTimer(5 * time.Second)
```

Comment:
Are there cases where 3 seconds wasn't enough?

Reply:
The test is somtimes failed with a timeout error. I tried locally, but it doesn't happen. I added some logs to check if handling logic takes time.